dot/state/service.go

```diff
@@ -34,6 +34,7 @@ type Service struct {
 	Transaction *TransactionState
 	Epoch       *EpochState
 	Grandpa     *GrandpaState
+	Slot        *SlotState
```

Comment:
Should slot state be a part of babe state?

Reply:
the slot state is a dependency for the BABE verifier, which is part of the BABE service but I don't think it should be part of the BABE state as in substrate they have a module only for slots (`client/consensus/slots`)